src/golang/lib/job/spec.go

```diff
@@ -82,10 +82,11 @@ type WorkflowRetentionSpec struct {
 
 type WorkflowSpec struct {
 	BaseSpec
-	WorkflowId     string                `json:"workflow_id" yaml:"workflowId"`
-	GithubManager  github.ManagerConfig  `json:"github_manager" yaml:"github_manager"`
-	Parameters     map[string]string     `json:"parameters" yaml:"parameters"`
-	AqPath         string                `json:"aq_path" yaml:"aqPath"`
+	WorkflowId    string               `json:"workflow_id" yaml:"workflowId"`
+	GithubManager github.ManagerConfig `json:"github_manager" yaml:"github_manager"`
+	Parameters    map[string]string    `json:"parameters" yaml:"parameters"`
+	AqPath        string               `json:"aq_path" yaml:"aqPath"`
+	// TODO: Deprecate StorageConfig
```

Comment:
We can't remove this just yet, since existing workflow specs will still have this field for deserialization might fail. (I need to confirm if that is the case or not.)